spec/enumerate_it_spec.rb

```diff
@@ -248,6 +248,18 @@ class Polymorphic
     end
   end
 
+  context 'fetching hash from enumeration' do
+    before do
+      class SomeEnum < EnumerateIt::Base
+        associate_values a: '1', b: '2', c: '3'
+      end
+    end
+
+    it 'should return values' do
```

Comment:
Please don't use `should` 🙂
http://www.betterspecs.org/#should

```suggestion
    it 'returns a hash' do
```